### PR TITLE
Addition of tmenu's README + licensing disclaimer

### PR DESCRIPTION
**Pull request by gbgabo**
PR focused on creating a first README of tyaml + disclaimer of licensing on the scripts source code header

### CODE REVIEW

File: tyaml/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# `tyaml`
```
Comment (calebe94) [inline tyaml/README.md]: This file does not belongs in this PR.

File: tyaml/tyaml [hunk 2]
```diff
@@ -1,5 +1,20 @@
 #!/bin/bash 
 
+#This file is part of the TinyTools distribution (https://github.com/Calebe94/TinyTools).
```
Comment (calebe94) [inline tyaml/tyaml]: This change does not belong in this PR

**Review by calebe94 — CHANGES_REQUESTED** [review]
Has to remove tyaml's changes.

**Comment by gbgabo** [comment]
PR solves #4

**Review by calebe94 — APPROVED** [review]
Everything seems OK to me.

**Pull request merged**